src/Application.njs

```diff
@@ -16,11 +16,20 @@ class Application extends Nullstack {
     const { value } = await HelloPlugin.sayHello();
   }
 
+  retrieveSubscriptionTime() {
+    const monthlyChecked = document.getElementById("monthly").checked
+    if(monthlyChecked){
+      return 'instill.monthly'
+    }
+    return 'instill.yearly'
```

Comment:
pls add it to a constant

Reply:
the function?

Reply:
no ... instill.monthy and instill.yearly

Reply:
why though?

Reply:
it is a better code and readability is better but it is up to you and this is just a suggestion.